Applications/Utils/MeshEdit/moveMeshNodes.cpp

```diff
@@ -201,8 +209,7 @@ int main (int argc, char* argv[])
     }
     /**** add other keywords here ****/
 
-    MeshLib::IO::writeMeshToFile(
-        *mesh, msh_name.substr(0, msh_name.length() - 4) + "_new.msh");
+    MeshLib::IO::VtuInterface vtu (mesh.get(), 0, false);
+    vtu.writeToFile(msh_name.substr(0, msh_name.length() - 4) + "_new.vtu");
```

Comment:
Just confirming: the 'msh' output is no longer needed? Otherwise there is writeMeshToFile() which is writing either 'vtu' or 'msh' depending on the extension, and is more general.